src/postgres/src/backend/catalog/yb_system_views.sql

```diff
@@ -1035,6 +1035,11 @@ CREATE VIEW tserver_stat_activity AS
     SELECT *
     FROM tserver_stat_get_activity();
 
+CREATE VIEW ycql_stat_statements AS
+    SELECT *
+    FROM ycql_stat_statements();
+REVOKE ALL ON ycql_stat_statements FROM public;
```

Comment:
What is the revoke/permission supposed to do? 

tserver_stat_activity doesn't seem to have an equivalent of this, although some others do.